selector-component.js

```diff
@@ -17,9 +17,11 @@ function register(module) {
       selected: '<brSelected',
       onSelect: '&brOnSelect',
       onAddItem: '&?brOnAddItem',
+      onDeselect: '&?brOnDeselect',
```

Comment:
onSelect handlers get transcluded into this module, so the bedrock-angular-selector itself isn't actually responsible for calling onSelect.